yt/yt/server/master/chunk_server/config.h

```diff
@@ -586,6 +586,9 @@ class TDynamicChunkManagerConfig
     //! Deprecated codec names and their alises, used values from yt/core/compression by default.
     std::optional<THashMap<TString, TString>> DeprecatedCodecNameToAlias;
 
+    //! Forbidden erasure codec ids, empty by default.
+    std::optional<THashSet<NErasure::ECodec>> ForbiddenErasureCodecIds;
```

Comment:
There is no need for std::optional here. THashSet is empty by default which is the desired behavior.